developing-grr/setting-up-dev-env.md

```diff
@@ -14,27 +14,27 @@ cd grr/
 
 ## Step 2. Set up Virtual Python environment.
 
-Make sure that you have `virtualenv` installed. It should be available in
+Make sure that you have `venv` installed. It should be available in
 repositories of all popular Linux distributions. For example, to install it on
 Ubuntu-based distros, run:
 
 ```bash
-sudo apt install virtualenv
+sudo apt install python3-venv
 ```
 
-To create a virtual environment you execute the `virtualenv $DIR` command
+To create a virtual environment you execute the `python3 venv $DIR` command
 where `$DIR` is the directory where you want it to be placed. The rest of the
-manual will assume that the environment is created in `~/.virtualenv/GRR` like
+manual will assume that the environment is created in `~/GRR` like
 this:
 
 ```bash
-virtualenv ~/.virtualenv/GRR --python=python3
+python3 -m venv ~/GRR
```

Comment:
Minor comment: let's not call the venv folder `GRR`. Since now we use `GRR` for venv and `grr` for the sources - it might be confusing. Should we rename `GRR` to `grr_venv`?

Reply:
SGTM!